drivers/filesystems/npfs/volinfo.c

```diff
@@ -78,7 +74,6 @@ NpQueryFsDeviceInfo(IN PVOID Buffer,
     PFILE_FS_DEVICE_INFORMATION InfoBuffer = Buffer;
     TRACE("Entered\n");
 
-    InfoBuffer->DeviceType = 0;
     InfoBuffer->Characteristics = 0;
```

Comment:
If `InfoBuffer->DeviceType` assignment is removed, this one should be removed as well.

Reply:
Thank you. I fixed.